netstandard/ref/System.Xml.Serialization.cs

```diff
@@ -47,6 +47,7 @@ public partial class ImportContext
     }
     public partial interface IXmlSerializable
     {
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
```

Comment:
Should not be exposed

Reply:
I think this one is intentional because we were trying to hide the dependency between IXmlSerializable and XmlSchema as we don't use it but we cannot remove it because this is an interface.

Reply:
Indeed.